### ISSUE TITLE
Improve mobile navigation experience

### ISSUE THREAD

**Issue by smoyth** [opendatakit/website#87]
The mobile version currently looks like this:

![image](https://user-images.githubusercontent.com/702320/40385405-799bfa10-5dd4-11e8-929b-7e0d2f74c46a.png)

It really needs some love. I think the easiest thing would be:

1. Get rid of the intra page links (there is a separate issue for this)
2. Move all navigational links into the hamburger menu (including _all_ the header links _and_ the side nav ones, probably in a tree layout).

Then you'd just have a clean page header with the hamburger followed by the title and content.

**Comment by wbrunette** [comment]
I agree with @smoyth that improving the mobile web viewing experience is important.
